pkgs/applications/misc/rofi/default.nix

```diff
@@ -19,11 +19,16 @@ stdenv.mkDerivation rec {
     sed -i 's/~root/~nobody/g' test/helper-expand.c
   '';
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+  nativeBuildInputs = [ autoreconfHook pkgconfig makeWrapper ];
   buildInputs = [ libxkbcommon pango cairo git bison flex librsvg check
     libstartup_notification libxcb xcbutil xcbutilwm xcbutilxrm which
   ];
 
+  postInstall = ''
+    wrapProgram $out/bin/rofi-theme-selector \
+      --set XDG_DATA_DIRS $out/share
```

Comment:
Any reason to not prefix XDG_DATA_DIRS (`--prefix XDG_DATA_DIRS : $out/share`)?

Reply:
:+1: This should be `--prefix`, It's very likely it discovers themes from `XDG_DATA_DIRS` so it won't be able to access others outside the package.

Reply:
Thanks for the argument. I hadn't thought of themes from other packages.